lib/matplotlib/__init__.py

```diff
@@ -121,19 +121,21 @@
 import subprocess
 import sys
 import tempfile
-import warnings
 
-import numpy
```

Comment:
Although these can potentially break existing code, it seems quite incorrect to import `warnings` or `numpy` from `matplotlib`.